objects/app/apps.go

```diff
@@ -148,7 +151,20 @@ func (a *appsCmd) create(c *cli.Context) error {
 	}
 
 	appWithFlags(c, app)
+	// If architectures flag is not set then default it to nil
+	if c.IsSet(SHAPE_PARAMETER) {
+		shapeParam := c.String(SHAPE_PARAMETER)
+
+		// Check for architectures parameter passed or set to default
+		if len(shapeParam) == 0 {
```

Comment:
As we are initialising the shapeParam before, is this check relevant?

Reply:
This is for the check if param is used but no value is passed. I can check if we can make it work without this check.